DirectXTex/DirectXTex.h

```diff
@@ -613,6 +613,12 @@ namespace DirectX
     constexpr unsigned long TEX_FILTER_MODE_MASK = 0xF00000;
     constexpr unsigned long TEX_FILTER_SRGB_MASK = 0xF000000;
 
+    struct ConvertOptions
```

Comment:
Please line up the fields in this struct like you did for ``CompressOptions``.

Reply:
Done.